samples/keys/identity_key_generation/README.rst

```diff
@@ -0,0 +1,71 @@
+.. _identity_key_generate:
+
+Identity key generation
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to generate a random device-specific identity key and store in the Key
+Management Unit (KMU). The identity key is stored in the KMU in encrypted form using the Hardware
+Unique Key (HUK) Master Key Encryption Key (MKEK). The sample also demonstrates how to generate a
+random MKEK and store it in KMU.
```

Comment:
Now we have two different overview sections in my eyes.

Can you merge everything except the first sentence into the overview below?

Reply:
Done, with minor alteration where reference to lib_hw_unique_key is moved to requirements